st2web/Dockerfile

```
@@ -78,5 +80,8 @@ VOLUME ["/etc/ssl/st2/"]
 
 EXPOSE 80
 EXPOSE 443
+EXPOSE 8080
+EXPOSE 8443
 STOPSIGNAL SIGTERM
-CMD ["/bin/bash", "-c", "if [ ${ST2WEB_HTTPS} = 1 ]; then ST2WEB_TEMPLATE='/etc/nginx/conf.d/st2-https.template'; else ST2WEB_TEMPLATE='/etc/nginx/conf.d/st2-http.template'; fi && envsubst '${ST2_AUTH_URL} ${ST2_API_URL} ${ST2_STREAM_URL}' < ${ST2WEB_TEMPLATE} > /etc/nginx/conf.d/st2.conf && exec nginx -g 'daemon off;'"]
+RUN touch /var/run/nginx.pid && chown 999:999 /etc/nginx/conf.d/ /var/cache/nginx/ /var/run/nginx.pid
```

```suggestion
RUN touch /var/run/nginx.pid && chown nginx:nginx /etc/nginx/conf.d/ /var/cache/nginx/ /var/run/nginx.pid
```

The nginx deb package does not hard-code a UID or GID, it only uses the user/group name when adding them, so should we do the same to avoid changes like you discovered with `101` changing to `999`.